.github/workflows/playwright.yml

```
@@ -0,0 +1,53 @@
+# todo: Uncomment later
```

actionlint (1.7.9)

workflow is empty

(syntax-check)